data/gui/custom_video_settings.stkgui

```diff
@@ -81,7 +81,7 @@
             <div layout="horizontal-row" proportion="1" height="fit">
                 <checkbox id="glow"/>
                 <spacer width="10" height="10"/>
-                <label text="Glow (outlines)" I18N="Video settings"/>
+                <label text="Glow (Outlines)" I18N="Video settings"/>
```

Comment:
I don't believe words inside parentheses require a capital letter? Got a reference for this? I found this ( http://englishplus.com/grammar/00000123.htm ) which seems to indicate the lower case was correct

Reply:
It depends whether or not they are separate sentences. Is it "Glow. Outlines." or "Glow outlines"? I don't know to be honest. I was just capitalising them as if they were separate.

Reply:
The intent, I guess, is "Glowing, sometime also referred to as outlines" or something like that